xonsh/xoreutils/which.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import builtins
+import functools
+import os
+
+from xonsh.xoreutils import _which
+from xonsh.platform import ON_WINDOWS, scandir
```

Comment:
I think this should be `import xonsh.platform as xp` to be consistent with other subpackages.